### PR TITLE
Tone pools bug fix

### PR DESCRIPTION
**Pull request by jdtoombs**
State was out of line with tone pools

### CODE REVIEW

File: app/editor/src/features/content/form/CondensedContentForm.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const CondensedContentForm: React.FC<ICondensedContentFormProps> = ({
   });
 
   const handleSubmit = async (values: IContentForm) => {
-    var contentResult: IContentModel | null = null;
+    let contentResult: IContentModel | null = null;
```
Comment (jdtoombs) [inline app/editor/src/features/content/form/CondensedContentForm.tsx:108]: linting warning about using var

Comment (Fosol) [reply app/editor/src/features/content/form/CondensedContentForm.tsx:108]: If the value never changes use `const`

File: app/editor/src/features/content/form/ContentForm.tsx [hunk 2]
```diff
@@ -126,9 +126,9 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType = Content
       // Now upload the file if it exists.
       if (!!values.file) {
         const uploadResult = await upload(contentResult, values.file);
-        setContent(toForm(uploadResult));
+        setContent(toForm({ ...uploadResult, tonePools: values.tonePools }));
```
Comment (Fosol) [inline app/editor/src/features/content/form/ContentForm.tsx:129]: I assume there is an issue with arrays or something here.  Can you explain?

File: app/editor/src/features/content/form/CondensedContentForm.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/features/content/form/CondensedContentForm.tsx:108]: Changes on line 124

File: app/editor/src/features/content/form/ContentForm.tsx (hunk 2 above)
Comment (jdtoombs) [reply app/editor/src/features/content/form/ContentForm.tsx:129]: For some reason when no tone values were assigned to an existing piece of content the array was having issues being assigned the new value into state. Didn't spend too much time on alternative methods to solve this; could look into more if need be

**Review by Fosol — APPROVED** [review]

**Pull request merged**